tooling/cli/index.bin.ts

```diff
@@ -139,5 +139,5 @@ const run = async (command: Command) => {
 };
 
 run(parseArgs())
-  .then(() => process.exit(0))
+  .then(() => process.exit(1))
```

Comment:
`1` is exit code for `fail/error`, this not represented the program run successfully  when put exit code `1`. 

see https://shapeshed.com/unix-exit-codes/